docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-source.md

```diff
@@ -0,0 +1,107 @@
+---
+id: taxii-2-source
+title: TAXII 2 Client
+sidebar_label: TAXII 2 Client
```

Comment:
```suggestion
sidebar_label: STIX TAXII 2.0 Client
```

Reply:
Any change here? They look the same to me.